### DISCUSSION TITLE
combination with a control desk

### DISCUSSION THREAD

**Discussion by Erhie** [easytarget/esp32-cam-webserver#255] [Ideas]
Hello!
Now I think about how to combine the video view with a control desk.
Now, my control desk page communicate with ESP board by fetching page with parameters.
By clicking following views  should be possible :
  video is the background of the control desk page.
  video is shown and can be adjusted like in this project.
 
Have you some a idea about?
Where to read about of the principle of the software interplay used in this project?

Thank you for discussion

**Comment by easytarget** [comment]
Sorry; I am not doing any further development on this project. it is old code and somewhat obsolete. I am currently working on retiring it; I will try to link to a more actively updated fork when I do.
